eng/common/TestResources/New-TestResources.ps1

```diff
@@ -270,6 +270,11 @@ try {
             '4d042dc6-fe17-4698-a23f-ec6a8d1e98f4' = 'Azure SDK Test Resources - TME '
         }
 
+        $wellKnownTenantTMEStatus = @{
+            '70a036f6-8e4d-4615-bad6-149c02e7720d' = $true  # tme testing
```

Comment:
I would invert this bool to make it clearer and match the usages of supportsSafeSecretStandard

Reply:
I'm not too pleased with the inverted conditional I added here. The tradeoff I considered when setting it this way is that we may have additional template conditionals based on TME status, so I wanted a simple hash that marked whether a tenant was TME or not. That way we could base other `supportsXYZStandard` off this. Given that, naming it the opposite `wellKnownTenantNonTMEStatus` seemed a bit odd.

I'm open to either approach here.

Reply:
If you just want this to be well-known TME subs then I would just make it a list of only the TME tenants and contain nothing else. So, it only answers the question of is this TME or not.